### ISSUE TITLE
Image detection/tracking not working on Android 10

### ISSUE THREAD

**Issue by DavidArayan** [google-ar/arcore-android-sdk#893]
We have tested using ARCore 1.12.0 on Google Pixel 2 and 3. Images fail to pickup/detected. The same code using ARCore 1.12.0 on Google Pixel 1 and other Android 9 devices works fine. Unfortunately we use our own pipeline so can't grab source examples. This is just standard image detection/tracking.

Logcat does not produce anything meaningful when either adding the images to the database or when attempting to detect an image. 

So to summarise, 

1. Image detection/tracking fails on Android 10 Pixel 2 and 3 using both ARCore 1.11.0 and 1.12.0.
2. image detection/tracking works on Android 9 Pixel 1, 2 and 3 using both ARCore 1.11.0 and 1.12.0

Same code, compiled using the same process (other than switching ARCore versions).

I will report if I find something new while investigating this.

Thank you!

**Comment by DavidArayan** [comment]
Hey guys,

Upon further investigation this had nothing to do with ARCore, but rather how we load/save images in our app. Android 10 has changed the way files are stored and permissions required for doing so. Switching to the Context.getExternalFilesDir() for storing our images solved our issue.

Apologies for the false alarm!
